test/powershell/Modules/Microsoft.PowerShell.Utility/clixml.tests.ps1

```diff
@@ -123,6 +123,58 @@
         }
     }
 
+    Context "ConvertTo-CliXml" {
+        BeforeAll {
+            $gpsList = Get-Process pwsh
+            $gps = $gpsList | Select-Object -First 1
+            $filePath = Join-Path $subFilePath 'gps.xml'
+        }
+
+        AfterEach {
+            Remove-Item $filePath -Force -ErrorAction SilentlyContinue
+        }
+
+        It "converts an object to string as expected" {
+            $objClixml = ConvertTo-Clixml -Depth 1 -InputObject ($gpsList | Select-Object -First 1)
+
+            foreach($item in $objClixml)
+            {
+                foreach($gpsItem in $gpsList)
+                {
+                    $checkId = $gpsItem.Id
+                    if (($null -ne $(Select-String -InputObject $item -SimpleMatch $checkId)) -and ($null -ne $(Select-String -InputObject $item -SimpleMatch "Id")))
+                    {
+                        $isExisted = $true
+                        break;
+                    }
+                }
+            }
+
+            $isExisted | Should Be $true
```

Comment:
Typically the error message on console when a failure happens with assertions like this are not helpful. 
`Was false when true was expected`.